stdlib/private/StdlibUnittest/RaceTest.swift

```diff
@@ -391,6 +392,7 @@ class _RaceTestWorkerState<RT : RaceTestWithPerTrialData> {
 
 class _RaceTestSharedState<RT : RaceTestWithPerTrialData> {
   var racingThreadCount: Int
+  var stopNow = false
```

Comment:
Yes in principle, although in practice it doesn't matter. I should add _stdlib_AtomicBool because this is the second place that wants it.